applovin_max/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Fix preloaded AdView not resuming auto-refresh on iOS.
```

Comment:
Can we reference the same thing consistently through the changelog? We reference it as `preloaded banners and MRECs` below
```suggestion
* Fix preloaded banners and MRECs not resuming auto-refresh on iOS.
```